config/base/common/network_policy.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: default-deny-all-traffic
+  namespace: system
+spec:
+  podSelector: { }
+  policyTypes:
+    - Ingress
+    - Egress
```

Comment:
Nit: Can we give a space at the end of the file ? 
See:

![Screenshot 2025-05-22 at 07 35 35](https://github.com/user-attachments/assets/5149026c-681e-498a-aa8b-35155a304097)